x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/__test__/ServiceOverview.test.tsx

```diff
@@ -65,53 +73,49 @@ describe('Service Overview -> View', () => {
 
   it('should render services, when list is not empty', async () => {
     // mock rest requests
-    const dataFetchingSpy = jest
-      .spyOn(callApmApi, 'callApmApi')
-      .mockResolvedValue({
-        hasLegacyData: false,
-        hasHistoricalData: true,
-        items: [
-          {
-            serviceName: 'My Python Service',
-            agentName: 'python',
-            transactionsPerMinute: 100,
-            errorsPerMinute: 200,
-            avgResponseTime: 300,
-            environments: ['test', 'dev']
-          },
-          {
-            serviceName: 'My Go Service',
-            agentName: 'go',
-            transactionsPerMinute: 400,
-            errorsPerMinute: 500,
-            avgResponseTime: 600,
-            environments: []
-          }
-        ]
-      });
+    coreMock.http.get.mockResolvedValueOnce({
+      hasLegacyData: false,
+      hasHistoricalData: true,
+      items: [
+        {
+          serviceName: 'My Python Service',
+          agentName: 'python',
+          transactionsPerMinute: 100,
+          errorsPerMinute: 200,
+          avgResponseTime: 300,
+          environments: ['test', 'dev']
+        },
+        {
+          serviceName: 'My Go Service',
+          agentName: 'go',
+          transactionsPerMinute: 400,
+          errorsPerMinute: 500,
+          avgResponseTime: 600,
+          environments: []
+        }
+      ]
+    } as never);
```

Comment:
why `never`?

Reply:
For some reason TS expects `never` as a type. I'm probably doing something wrong, but I couldn't figure it out and didn't want to spend too much time fixing types for a test. Do you happen to know what's going on?

Reply:
What do you think about:

```ts
    function getCoreMock(res: HttpBody) {
      return ({
        http: {
          basePath: {
            prepend: (path: string) => `/basepath${path}`
          },
          get: jest.fn(res)
        },
        notifications: {
          toasts: {
            addWarning: () => {}
          }
        }
      } as unknown) as LegacyCoreStart & {
        http: { get: jest.SpyInstance<LegacyCoreStart['http']['get'], any> };
      };
    }
```

This way the spies are not modified, and the compile error disappears

Reply:
I think my type was just wrong. Simplified to `{ get: jest.Mock<any,any> }` and that works as well.

Reply:
👍